### ISSUE TITLE
[Feature] Create `tasks.json` file while creating new Clarinet project.

### ISSUE THREAD

**Issue by LNow** [hirosystems/clarinet#154]
While creating new project Clarinet should create `tasks.json` file next to `settings.json` fine in `.vscode` directory with two predefined tasks. Firs one to run `clarinet check` and second one `clarinet test`.

It could have following content:
```json
{
  "version": "2.0.0",
  "tasks": [
    {
      "label": "check contracts",
      "group": "test",
      "type": "shell",
      "command": "clarinet check"
    },
    {
      "label": "test contracts",
      "group": "test",
      "type": "shell",
      "command": "clarinet test"
    }
  ]
}
```

**Comment by lgalabru** [comment]
That'd be great! Want to take a stab at it? 🤗

**Comment by LNow** [comment]
I'll try :)

**Comment by lgalabru** [comment]
Addressed with #158
